example/src/main.rs

```diff
@@ -2,16 +2,20 @@
 mod ffi {
     use fixed_decimal::FixedDecimal;
 
-    pub struct ICU4XFixedDecimal(pub FixedDecimal);
+    pub struct ICU4XFixedDecimal(pub Box<FixedDecimal>);
```

Comment:
I actually don't think we should be doing this, because this adds an extra layer of indirection for the `&self` methods.

I'd do #[repr(transparent)] pub struct ICU4XFixedDecimal(pub FixedDecimal);` and have `new()` return a `Box<ICU4XFixedDecimal>`